src/scp/BallotProtocol.cpp

```diff
@@ -242,10 +242,17 @@ bool
 BallotProtocol::isStatementSane(SCPStatement const& st, bool self)
 {
     auto qSet = mSlot.getQuorumSetFromStatement(st);
-    bool res = qSet != nullptr && isQuorumSetSane(*qSet, false);
+    const char* errString = nullptr;
+    bool res = qSet != nullptr && isQuorumSetSane(*qSet, false, errString);
     if (!res)
     {
         CLOG(DEBUG, "SCP") << "Invalid quorum set received";
+        if (errString != nullptr)
+        {
+            std::string msg(errString);
+            CLOG(DEBUG, "SCP") << msg;
+        }
```

Comment:
Can someone suggest a way to inline this into the previous `CLOG` call?

Reply:
I would just write this as
`CLOG(DEBUG, "SCP") << "Invalid quorum set received : " << (errString?errString:"<empty>");` and be done with it (also avoids a new variable).